test/entry_test.rb

```diff
@@ -40,42 +41,20 @@ def test_is_directory_and_is_file
 
   def test_equality
     entry1 = ::Zip::Entry.new('file.zip', 'name', 'isNotCompared',
-                              'something extra', 123, 1234,
-                              ::Zip::Entry::DEFLATED, 10_000)
+                              ::Zip::Entry::DEFLATED, ::Zip.default_compression)
     entry2 = ::Zip::Entry.new('file.zip', 'name', 'isNotComparedXXX',
-                              'something extra', 123, 1234,
-                              ::Zip::Entry::DEFLATED, 10_000)
+                              ::Zip::Entry::DEFLATED, ::Zip.default_compression)
     entry3 = ::Zip::Entry.new('file.zip', 'name2', 'isNotComparedXXX',
-                              'something extra', 123, 1234,
-                              ::Zip::Entry::DEFLATED, 10_000)
+                              ::Zip::Entry::DEFLATED, ::Zip.default_compression)
     entry4 = ::Zip::Entry.new('file.zip', 'name2', 'isNotComparedXXX',
-                              'something extraXX', 123, 1234,
-                              ::Zip::Entry::DEFLATED, 10_000)
-    entry5 = ::Zip::Entry.new('file.zip', 'name2', 'isNotComparedXXX',
```

Comment:
If I understand the test correctly, `entry4 != entry5` is checking that the comparison is based on the compressed size. While setting up `entry5` would now require setting the compressed size post-construction, I think it's still a valid test to do? (And so on for entries 6-8.)

Reply:
Again, these just revert too.